authentication/models.py

```diff
@@ -98,6 +98,9 @@ class Company(models.Model):
     startup_idea = models.TextField(blank=True)
     tags = models.CharField(max_length=255, blank=True)
 
+    def __str__(self):
+        return self.brand
+
 
 class CompanyAndUserRelation(models.Model):
     FOUNDER = "F"
```

Comment:
class CompanyAndUserRelation(models.Model):
    FOUNDER = "F"

class UserLoginActivity(models.Model):
    # Login Status
    FAILED = 'F'
Using the same variables in the same file is a bad practice.